src/main/java/uk/gov/hmcts/reform/sscs/service/hmc/topic/ProcessHmcMessageService.java

```diff
@@ -51,7 +51,9 @@ public void processEventMessage(HmcMessage hmcMessage)
 
         checkStatuses(caseId, hearingId, hmcMessageStatus, hmcStatus);
 
-        if (stateNotHandled(hmcStatus, hearingResponse)) {
+        boolean isHearingUpdated = isHearingUpdated(hmcStatus, hearingResponse);
```

Comment:
Just a suggestion, could it be final? Or it is not necessary?

Reply:
No reason, why not. Added